appeals/web/src/server/lib/mappers/data/types.d.ts

```diff
@@ -0,0 +1,16 @@
+import { MappedInstructions } from '../types';
+
+export type InitialiseAndMapDataFactory = <
+	TInnerParams extends Record<string, unknown>,
```

Comment:
What's the reason for the `T` prefix here? I'd have thought this could just refer to the `InnerParams` type defined in TSDoc further up

Reply:
It's a generic, just a convention